limiter/request_limiter.go

```diff
@@ -0,0 +1,146 @@
+package limiter
```

Comment:
By convention, in this project, we call the request rate limiter a "limiter". Please, rename the package to something else, e.g. "blocker" or something.

Reply:
Renamed package to blocker